src/transformers/utils/dummy_pt_objects.py

```diff
@@ -1755,6 +1755,34 @@ def __init__(self, *args, **kwargs):
         requires_backends(self, ["torch"])
 
 
+CPMANT_PRETRAINED_MODEL_ARCHIVE_LIST = None
+
+
+class CPMAntForCausalLM(metaclass=DummyObject):
+    _backends = ["torch"]
+
+    def __init__(self, *args, **kwargs):
+        requires_backends(self, ["torch"])
+
+
+class CPMAntModel(metaclass=DummyObject):
+    _backends = ["torch"]
+
+    def __init__(self, *args, **kwargs):
+        requires_backends(self, ["torch"])
+
+
+class CPMAntPreTrainedModel(metaclass=DummyObject):
+    _backends = ["torch"]
+
+    def __init__(self, *args, **kwargs):
+        requires_backends(self, ["torch"])
```

Comment:
```suggestion
    _backends = ["torch", "jieba"]

    def __init__(self, *args, **kwargs):
        requires_backends(self, ["torch", "jieba"])
```

Reply:
As I am not sure here, please wait for the review of @sgugger  🙏

Reply:
The model has no requirement on jieba.